tests/features/event.feature

```diff
@@ -57,6 +57,8 @@ Feature: Event content creation
 
```

Comment:
Locally I have an error in this test because I have two medias "Euro with miniature figurines". I've fixed it locally by removing following code:
```
    # Create a "Media AV portal photo".
    And the following AV Portal photos:
      | url                                                         |
      | https://audiovisual.ec.europa.eu/en/photo/P-038924~2F00-15  |
      | https://audiovisual.ec.europa.eu/en/photo/P-039321~2F00-04  |

    # Create a "Media AV portal photo".
    And the following AV Portal photos:
      | url                                                         |
      | https://audiovisual.ec.europa.eu/en/photo/P-038924~2F00-15  |
```
as I see there is Drupal\Tests\oe_media\Behat\AvPortalContext where `media_avportal_mock` module is enabled. This module also provides "Euro with miniature figurines" media image too.

Could you please try to run this test locally. Do you face with issue?